basis/ui/tools/tools.factor

```diff
@@ -1,13 +1,16 @@
 ! Copyright (C) 2006, 2009 Slava Pestov.
 ! See http://factorcode.org/license.txt for BSD license.
-USING: memory system kernel vocabs.refresh ui.tools.operations
-ui.tools.listener ui.tools.browser ui.tools.common ui.tools.error-list
-ui.tools.walker ui.commands ui.gestures ui ui.private ;
+USING: kernel listener memory system ui ui.commands ui.gestures
+ui.private ui.tools.browser ui.tools.common ui.tools.error-list
+ui.tools.listener ui.tools.operations ui.tools.walker
+vocabs.refresh ;
 IN: ui.tools
 
 MAIN: listener-window
 
-\ refresh-all H{ { +nullary+ t } { +listener+ t } } define-command
+: refresh-all-cmd ( -- ) [ refresh-all ] with-ctrl-break ;
+
+\ refresh-all-cmd H{ { +nullary+ t } { +listener+ t } } define-command
```

Comment:
Instead of doing this just for ``refresh-all``, maybe we should instead change the ``ui.operations`` code to do this for all commands?